### DISCUSSION TITLE
AFL++ on Coreutils

### DISCUSSION THREAD

**Discussion by arindam-8** [AFLplusplus/AFLplusplus#1970] [Q&A] [answered]
Hi, 

I am looking into using AFL++ for testing utilities like `cut`,`seq` etc. I do understand that there is _some_ stuff for argv fuzzing, but I am not quite sure of its capabilities (for instance for `cut` which takes both command line arguments and possibly a file). If my goal is to fuzz both the options and the file, how should I go about doing this? 

Thanks
Arindam

**Comment by vanhauser-thc** [comment, marked as answer]
Have you read the documentation ?

**Reply by arindam-8** [reply]
Thanks!

**Reply by arindam-8** [reply]
I'll take a look in more detail.
